README.md

```diff
@@ -80,79 +82,93 @@ The individual pdf-files are stored in `./visualisations/plots/trajectory`.\
->        ```bash
+>```bash
 >        sudo mkdir /mnt/e
->        ```
+>```
 >3. change the active working directory to 
->    ```bash 
+>```bash 
 >    cd ~/project21/fishproviz
->    ```
+>```
 >4. open the file `fishproviz/config.env` to configure the environment file and set the path variables that point to your working directory
->    ```bash
+>```bash
 >    notepad.exe fishproviz/config.env &
->    ```
+>```
 >    * please note, that linux doesn't accept whitespaces or special characters (like braces) in file names. These need to be escaped with a backslash
 >    * note that the prefix to the different data sources is `/mnt/e`.
 >5. for calculating trajectories, run 
->    ```bash 
+>```bash 
 >    python3 main.py trajectory
->    ```
+>```
 >6. bundling each trajectory file into one trajectory-file per individual using:
->    ```bash
+>```bash
 >    bash scripts/build-trajectories.sh -l #-l flag for using the local configurations
->    ```
+>```
 >   * if the execution of this bash-script results in permission errors, check the access-rights for this files using 
->   ```bash
+>```bash
 >    ls -las scripts/build-trajectories.sh
 >    # output: >>> 16 -rw-r--r--  1  user user-group date time scripts/build-trajectories.sh
 >    chmod u+x scripts/build-trajectories.sh # users receive execution-rights for this file
 >    ls -las scripts/build-trajectories.sh
 >    # output: >>> 16 -rwxr--r--  1  user user-group date time scripts/build-trajectories.sh
->    ```
+>```
 
 #### Plot Generation for Feeding Trajectories
 For generating single pdf-files with plots with trajectories for every timeframe, run the following command:
 -   Feeding Trajectories: `python3 main.py feeding`
         The CSV-file for time spend, feeding and number of visits are stored at `results/feeding`.
-    -   Optional requirement: Provide a csv-file (;-separated) with start and end time for the feeding measures with columns in the following format:
+-   Optional requirement: Provide a csv-file (;-separated) with start and end time for the feeding measures with columns in the following format:
 
-        | day | time_in_start | time_in_stop | time_out_start | time_out_stop |
-        |-----|---------------|--------------|----------------|---------------|
-        | dd.mm.yy| hh:mm | hh:mm | hh:mm | hh:mm |
+| day | time_in_start | time_in_stop | time_out_start | time_out_stop |
+|-----|---------------|--------------|----------------|---------------|
+| dd.mm.yy| hh:mm | hh:mm | hh:mm | hh:mm |
 
-    ```bash
+```bash
     # fishproviz/config.env
     SERVER_FEEDING_TIMES_FILE="path/to/feeding_times.csv"
-    FEEDING_SHAPE="ellipse" # or "patch", "rectangle"
-    ```
+    FEEDING_SHAPE="ellipse" # or "patch"
+    FEEDING_SHAPE_WIDTH=5
+    FEEDING_SHAPE_HEIGHT=5
+    MAGNET_LENGTH_CM=2.5
```

Comment:
while I see an intrinsic motivation of defining the magnet length as well in the config, it might be redundant or even convoluting in practice - see review for fishproviz/trajectory/feeding_shape.py